### PR TITLE
Marionette.noConflict Documentation

### PR DESCRIPTION
**Pull request by aeksco**
### Proposed changes
 - Docs for Marionette.noConflict()
 - Docs for Marionette.VERSION
 - Docs for Marionette.DEV_MODE

Link to the issue: [#3249 - Document noConflict method](https://github.com/marionettejs/backbone.marionette/issues/3249)

### CODE REVIEW

File: docs/marionette.configuration.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+* [Marionette.DEV_MODE](#marionettedevmode)
+
+## Marionette.VERSION
+
+Maintains a reference to the version of a Marionette instance. Marionette.VERSION is used by unit tests to direct users to the correctly versioned documentation when errors are thrown.
+
+
+## Marionette.DEV_MODE
+
+A boolean flag (default is ```false```) used by unit tests to handle calls to ```Marionette.deprecate()```. If the flag is set to ```true```, deprecation console warnings are issued at runtime.
```
Comment (paulfalgout) [inline docs/marionette.configuration.md]: Inline code can just use single \`s  You only need 3 for multiline.

Comment (aeksco) [inline docs/marionette.configuration.md]: Nice - thanks for the tip! I'll adjust the PR accordingly :)

File: docs/marionette.functions.md [hunk 2]
```diff
@@ -460,3 +461,17 @@ myView.do('action:two');
...
+
+Allows you to run multiple instances of Marionette in the same application. After loading the new version, call `noConflict()` to get a reference to it. At the same time the old version will be returned to Backbone.Marionette.
+
+```javascript
+var Mn = require('backbone.marionette');
+
+var MnV3 = require('backbone.marionette/v3');
+
+// Creates non-conflicting reference to secondary Marionette instance
+MnNew = MnNew.noConflict()
```
Comment (scott-w) [inline docs/marionette.functions.md]: Should this be `Mn.noConflict()` or `MnV3.noConflict?`?

Comment (aeksco) [inline docs/marionette.functions.md]: Yes it should be - nice catch. Last-minute rename on my end. Updates incoming...

**Comment by scott-w** [comment]
@aeksco That's good to go. Can you just ensure the rebase conflicts are resolved and we'll be happy to get it in 👍

**Pull request merged**